### PR TITLE
Accept equivalent tar ball paths

### LINKED ISSUE #89: False positive `Missing npm packages:` when installing from tarball
I've run into a problem that I suspect is related to `ember-cli-dependency-checker`:
https://github.com/felixrieseberg/ember-electron/pull/355#issuecomment-404622536

Basically we are installing a package via the [`npm install <tarball file>` syntax](https://docs.npmjs.com/cli/install) (to work around a caching problem and a symlink problem), but the dependency checker seems to get confused about this and think the wrong package is installed: 

```
Missing npm packages: 
Package: ember-electron
  * Specified: file:../../tmp-2580rXGPRrApjWO8/ember-electron-cachebust.tgz
  * Installed: file:C:/Users/appveyor/AppData/Local/Temp/1/tmp-2580rXGPRrApjWO8/ember-electron-cachebust.tgz
Run `npm install` to install missing dependencies.
```

### PR DESCRIPTION
**Pull request by jacobq**
Closes #89

### CODE REVIEW

**Comment by stopdropandrew** [comment]
This was a hell of a bug to track down!

File: lib/version-checker.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 'use strict';
+const path = require('path');
```
Comment (quaertym) [inline lib/version-checker.js]: We should move this to where it is used.

File: lib/version-checker.js [hunk 2]
```diff
@@ -25,7 +26,7 @@ VersionChecker.satisfies = function(versionSpecified, versionInstalled) {
       }
     }
   } else if (isTarGz(version) && versionInstalled !== unknownVersion) {
-    return version === versionInstalled;
+    return (version === versionInstalled) || (path.resolve(version) === path.resolve(versionInstalled));
```
Comment (quaertym) [inline lib/version-checker.js]: We can remove the first part `(version === versionInstalled)`.

**Comment by quaertym** [comment]
Thanks! I added some comments then we are good to merge.

File: lib/version-checker.js (hunk 1 above)
Comment (jacobq) [reply lib/version-checker.js]: OK, I moved it all the way down into the `else if` block where it is needed.

**Pull request merged**